src/objc_glue.c

```diff
@@ -145,7 +145,9 @@ Symbol *objc_getMethVarName(const char *s, size_t len)
 
 Symbol *objc_getMethVarName(Identifier *ident)
 {
-    return objc_getMethVarName(ident->string, ident->len);
+    const char *id = ident->toChars();
+    size_t id_length = strlen(id);
+    return objc_getMethVarName(id, id_length);
```

Comment:
Why is this change necessary?

Reply:
https://auto-tester.puremagic.com/show-run.ghtml?projectid=17&runid=3396619&isPull=true

Whilst waiting, just curious if OSX builds would pass if the compiler error were fixed.

Reply:
I see now that this might be necessary since `Identifier` is now holding a D string.

Reply:
The fields in `Identifier` used to public at one point in time.